app/src/main/res/values-sw/strings.xml

```diff
@@ -1,272 +1,418 @@
+    <string name="delete_beneficiary">Ondoa mrithi</string>
+    <string name="delete_beneficiary_confirmation">Una uhakika unataka kuondoa mrithi huyu?</string>
+    <string name="delete">Kufuta</string>
+    <string name="beneficiary_deleted_successfully">Mrithi ameondolewa</string>
+    <string name="third_party_transfer">Uhamisho wa mtu wa tatu</string>
+    <string name="total_loan">Jumla ya mkopo</string>
+    <string name="total_saving">Uhifadhi wa jumla</string>
+    <string name="accounts_overview">Maelezo ya Akaunti</string>
+    <string name="survey">Surveys</string>
+    <string name="activation_date">Tarehe ya uanzishaji</string>
+    <string name="groups">Kundi</string>
+    <string name="user_details">Data ya mtumiaji</string>
+    <string name="client_type">Aina ya Wateja</string>
+    <string name="client_classification">Uainishaji wa Wateja</string>
+    <string name="select_mode">Chagua mode</string>
+    <string name="add_beneficiary_option">Ongeza warithi wa nje kwenye akaunti yako. \n Ingiza kwa manually au soma msimbo wa QR wa akaunti yako</string>
+    <string name="invalid_qr">Huwezi kufanya vitendo kwenye akaunti yako, soma msimbo wa QR kutoka akaunti ya akiba au akaunti ya mkopo wa mtumiaji mwingine</string>
+    <string name="add">Kuongeza</string>
+    <string name="scan">Scan</string>
+    <string name="enter_passcode">Ingiza msimbo wa kufikia tarakimu nne</string>
+    <string name="error_passcode">Nambari ya upatikanaji lazima iwe na tarakimu nne</string>
+    <string name="incorrect_passcode">Nambari ya ufikiaji isiyo sahihi</string>
+    <string name="incorrect_passcode_more_than_three">Umeingia msimbo usio sahihi wa zaidi ya mara 3</string>
+    <string name="skip">Ruka</string>
+    <string name="save">Save</string>
+    <string name="passcode_setup">Weka pini ili uingie</string>
+    <string name="reenter_passcode">Rejesha tena siri</string>
+    <string name="passcode_does_not_match">Msimbo wa kufikia haufanani.</string>
+    <string name="forgot_passcode">Nilisahau siri, nitaingia na nenosiri</string>
+    <string name="proceed">Kuendelea</string>
+    <string name="amount_greater_than_zero">Kiasi kinapaswa kuwa kikubwa kuliko sifuri</string>
+    <string name="about_us_under_construction">Bookmark Kuhusu sisi chini ya ujenzi</string>
+    <string name="help_under_construction">Msaada chini ya kichupo cha ujenzi</string>
+    <string name="logout">Ondoka kwenye</string>
+    <string name="share_msg">Pakua programu ya Self Service hapa: https://play.google.com/store/apps/details?id=</string>
+    <string name="choose">Chagua programu</string>
+    <string name="not_available">THE</string>
+    <string name="upload_qr_code">Weka QR</string>
+    <string name="select_region_qr">Chagua kanda na msimbo wa QR</string>
 
-    <!--Customer care details-->
-    <string name="need_help">need help? Contact your support line</string>
-    <string name="help_line_number">8000000000</string>
```

Comment:
Don't remove this string, keep it as it is.

Reply:
Okay